src/main/java/styluscar/material/MaterialResource.java

```diff
@@ -6,45 +6,44 @@
 
 import javax.inject.Inject;
 import javax.transaction.Transactional;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
 import java.util.List;
 
 @Path("material")
 public class MaterialResource {
 
     @Inject
-    MaterialService ms;
+    MaterialService service;
 
     @GET
     @Path("/materials")
+    @Produces(MediaType.APPLICATION_JSON)
     public List<Material> listAllMaterials(){
         try {
-            return this.ms.findAllMaterial();
+            List<Material> materials = this.service.findAllMaterial();
+            return materials;
```

Comment:
Use `return this.service.findAllMaterial();`

Reply:
done